### PR TITLE
Article about AdoptOpenJDK...

### PR DESCRIPTION
**Pull request by hendrikebbers**

### CODE REVIEW

File: _posts/2020-06-29-adoptopenjdk-eclipse.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  'AdoptOpenJDK becomes Eclipse Adoptium'
+author: hendrik
+featuredImage: comment
+excerpt: 'A few days ago, AdoptOpenJDK officially announced its accession to the Eclipse Foundation and the future name "Eclipse Adoptium". A necessary and sensible move? A comment by Hendrik Ebbers, member of the Technical Steering Committee of AdoptOpenJDK.'
```
Comment (hendrikebbers) [inline _posts/2020-06-29-adoptopenjdk-eclipse.md:6]:
```suggestion
excerpt: 'AdoptOpenJDK officially announced its accession to the Eclipse Foundation and the future name "Eclipse Adoptium". A necessary and sensible move? A comment by Hendrik Ebbers, member of the Technical Steering Committee of AdoptOpenJDK.' 
```

File: _posts/2020-06-29-adoptopenjdk-eclipse.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+author: hendrik
+featuredImage: comment
+excerpt: 'A few days ago, AdoptOpenJDK officially announced its accession to the Eclipse Foundation and the future name "Eclipse Adoptium". A necessary and sensible move? A comment by Hendrik Ebbers, member of the Technical Steering Committee of AdoptOpenJDK.' 
+categories: [Java, OpenJDK, Eclipse, Adoptium]
+header:
+  text: AdoptOpenJDK becomes <span class="my-karakun">Eclipse Adoptium</span>
+  image: post
+---
+
+AdoptOpenJDK has become one of the biggest Java respectively OpenJDK distributions with [more than 165M downloads](https://dash-v2.adoptopenjdk.net/). As a member of the Technical Steering Committee (TSC) of AdoptOpenJDK, I did not only follow the project in the last couple of months but was at least in pieces a vital part of it.
```
Comment (hendrikebbers) [inline _posts/2020-06-29-adoptopenjdk-eclipse.md:13]:
```suggestion
AdoptOpenJDK has become one of the biggest Java respectively OpenJDK distributions with [more than 170M downloads](https://dash-v2.adoptopenjdk.net/). As a member of the Technical Steering Committee (TSC) of AdoptOpenJDK, I did not only follow the project in the last couple of months but was at least in pieces a vital part of it.
```

File: _posts/2020-06-29-adoptopenjdk-eclipse.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Why AdoptOpenJDK?
+
+If I was asked 5 years ago about where to download Java I definitely would have referred to the download pages of the Oracle JDK. But in the last couple of years, things have changed. Thanks to OpenJDK, in theory everyone can build its own Java distribution. AND, licensing of Oracle JDK has changed - [it is no longer free to use](/java/2018/06/25/java-releases.html). This is why more and more Java distributions came up and it is sometimes not easy to get a clear overview of this JDK jungle.
```
Comment (hendrikebbers) [inline _posts/2020-06-29-adoptopenjdk-eclipse.md:17]: Why is the AND all Uppercase at the beginning of a new sentence?

File: _posts/2020-06-29-adoptopenjdk-eclipse.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+![3 Duke Suspects](/assets/images/pages/jdks/3duke_suspects.png)
+
+In contrast to OpenJDK distributions by companies like Oracle, IBM, Azul or Bellsoft, [AdoptOpenJDK](https://adoptopenjdk.net/) is a community-based project providing Java binaries for a large number of platforms. Besides the common binaries for Windows x64, Mac x64 and Linux x64 there are also builds for AIX and ARM.
+
```
Comment (hendrikebbers) [inline _posts/2020-06-29-adoptopenjdk-eclipse.md:22]: platforms
Image of platforms can be added

File: _posts/2020-06-29-adoptopenjdk-eclipse.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+One of the maybe biggest advantages of the provided builds is the Long-Term-Support (LTS). Unlike some commercial providers of Java binaries, AdoptOpenJDK delivers free updates with important security fixes for Java LTS versions (e.g. Java 8 and 11) even 6 months later.
```
Comment (hendrikebbers) [inline _posts/2020-06-29-adoptopenjdk-eclipse.md:23]: Links zu LTS: https://adoptopenjdk.net/support.html#roadmap

**Comment by hendrikebbers** [comment]
Maybe we can use this one as featured image: https://unsplash.com/photos/gOROUFiJ8qU

**Comment by kress04** [comment]
@hendrikebbers, I updated the .md-file. Can you please check again? Thx.

File: _posts/2020-06-29-adoptopenjdk-eclipse.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+![3 Duke Suspects](/assets/images/pages/jdks/3duke_suspects.png)
+
+In contrast to OpenJDK distributions by companies like Oracle, IBM, Azul or Bellsoft, [AdoptOpenJDK](https://adoptopenjdk.net/) is a community-based project providing Java binaries for a large number of platforms. Besides the common binaries for Windows x64, Mac x64 and Linux x64 there are also builds for AIX and ARM.
+
+![Platforms supported by AdoptOpenJDK](/assets/posts/2020-07-13-adoptium/platforms.png)
+
+One of the maybe biggest advantages of the provided builds is the [Long-Term-Support (LTS)](https://adoptopenjdk.net/support.html#roadmap). Unlike some commercial providers of Java binaries, AdoptOpenJDK delivers free updates with important security fixes for Java LTS versions (e.g. Java 8 and 11) even 6 months later.
```
Comment (hendrikebbers) [inline _posts/2020-06-29-adoptopenjdk-eclipse.md:25]:
```suggestion
One of the biggest advantages of the provided builds is the [Long-Term-Support (LTS)](https://adoptopenjdk.net/support.html#roadmap). Unlike some commercial providers, AdoptOpenJDK delivers free updates with important security fixes for Java LTS versions (e.g. Java 8 and 11) even after a period of 6 months.
```

**Pull request merged**